libs/formik-elements/src/RadioGroupField.tsx

```diff
@@ -30,14 +31,25 @@ type RadioGroupFieldItemProps = RadioGroupItemProps;
 function RadioGroupField({ name, children, ...props }: RadioGroupFieldProps) {
   const [field, meta, helpers] = useField(name);
   const shouldValidate = useShouldValidate();
+  const initialError = useFormikBypass(name);
+
+  const onChange = (value: string | boolean | null) => {
+    helpers.setValue(value, shouldValidate);
+    initialError.current = undefined;
```

Comment:
are we doing this everywhere or just here?

Reply:
Everywhere, on every field component